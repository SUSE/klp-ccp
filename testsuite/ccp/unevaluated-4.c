static int g(void)
{
	return 1;
}

static int h(void)
{
	return 1;
}

static int i(void)
{
	return 1;
}

static int j(void)
{
	return 1;
}

static int k(void)
{
	return 1;
}

static int l(void)
{
	return 1;
}

void pu_f(int x)
{
	0 || g();
	1 || h();
	x || i();

	0 && j();
	1 && k();
	x && l();
}
