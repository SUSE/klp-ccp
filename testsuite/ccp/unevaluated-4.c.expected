/* klp-ccp: from testsuite/ccp/unevaluated-4.c */
static int g(void)
{
	return 1;
}

static int h(void);

static int i(void)
{
	return 1;
}

static int j(void);

static int k(void)
{
	return 1;
}

static int l(void)
{
	return 1;
}

void klpp_pu_f(int x)
{
	0 || g();
	1 || h();
	x || i();

	0 && j();
	1 && k();
	x && l();
}
